docs/graphcast/radios/poi-radio.md

```diff
@@ -88,29 +144,6 @@ To have full control over the POI Radio code and run it directly on your machine
 cargo run
 ```
 
-## Configuration
-
-### Testnet (Goerli)
-
-| Name                         | Example                                                                   |
-| ---------------------------- | ------------------------------------------------------------------------- |
-| `ETH_NODE`                   | `https://goerli.infura.io/v3/<PROJECT_ID>`                                |
-| `PRIVATE_KEY`                | `0x0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef`      |
-| `GRAPH_NODE_STATUS_ENDPOINT` | `http://localhost:8030/graphql`                                         |
-| `REGISTRY_SUBGRAPH`          | `https://api.thegraph.com/subgraphs/name/hopeyen/gossip-registry-test`    |
-| `NETWORK_SUBGRAPH`           | `https://gateway.testnet.thegraph.com/network`                            |
-| `WAKU_HOST` (Optional)       | Defaults to `127.0.0.1`                                                   |
-| `WAKU_PORT` (Optional)       | Defaults to `8546`                                                        |
-| `SLACK_TOKEN` (Optional)     | `xoxp-0123456789-0123456789-0123456789-0123456789` (defaults to `None`)   |
-| `SLACK_WEBHOOK` (Optional)   | `https://hooks.slack.com/services/<ID>/<ID>/<TOKEN>` (defaults to `None`) |
-| `LOG_LEVEL` (Optional)       | Defaults to `INFO`                                                        |
-
-`SLACK_TOKEN` and `SLACK_WEBHOOK` are used for POI divergence notifications in a Slack channel.
-
-`WAKU_HOST` and `WAKU_PORT` specify where the Graphcast node (included in all Radios) runs. If you want to run multiple Radios, or multiple instances of the same Radio, you should run them on different ports. 
-
-If you want to customize the log level, you can set `LOG_LEVEL` to one of `ERROR`, `WARN`, `DEBUG`, `TRACE`.
-
 ## Workflow
```

Comment:
I think this section is due for an update, in particular
- periodic polling of active allocations, not just upon initiation
- radio has became independent of listening for new blocks on Ethereum, it queries graph node instead
- messages also contain `network` now